src/conf_mode/vpn_openconnect.py

```diff
@@ -39,6 +39,34 @@
 def get_hash(password):
     return crypt(password, mksalt(METHOD_SHA512))
 
+
+def T2665_default_dict_cleanup(origin: dict, default_values: dict) -> dict:
+    """
+    https://phabricator.vyos.net/T2665
+    Clear unnecessary key values in merged config by dict_merge function
+    :param origin: config
+    :type origin: dict
+    :param default_values: default values
+    :type default_values: dict
+    :return: merged dict
+    :rtype: dict
+    """
+
+    if 'mode' in origin["authentication"] and "radius" in \
+            origin["authentication"]["mode"]:
+        del origin['authentication']['radius']['server']['port']
+        if not origin["authentication"]['radius']['server']:
+            raise ConfigError(
+                'Openconnect mode radius required at least one radius server')
```

Comment:
I think it should be either `OpenConnect authentication mode is set to RADIUS but RADIUS servers are not configured` or `"vpn openconnect authentication mode radius" requires at least one RADIUS server`. Clarity of config errors is important, and so if their formatting.

Reply:
Changed to 'openconnect authentication mode radius requires at least one RADIUS server'